### ISSUE TITLE
Upgrade hpfeeds

### ISSUE THREAD

**Issue by wrharding** [wrharding/mhn#18]
The current hpfeeds install hasn't had a commit in 4 years and hasn't been actively maintained for 6+.

https://github.com/pwnlandia/hpfeeds

Merging the current fork with hpfeeds won't work as hpfeeds has moved from a honeypot-specific configuration to an agnostic approach. The commits added to the pwnlanida version are no longer relevant. 

We can attempt to utilize the most up to date version of the primary hpfeeds repo, but a lot of configuration will be needed.

https://github.com/hpfeeds/hpfeeds

**Comment by wrharding** [comment]
Docs:

https://python.hpfeeds.org/en/latest/index.html

Website:

https://hpfeeds.org/

**Comment by wrharding** [comment]
After some fiddling I got hpf3 working after installing from scripts/install_hpfeeds.sh in 1223a75.

Success after

Upgrading `pyOpenssel` to v23.1.1 (latest as of writing). Note, pyOpenssl states Python 3.6 support will be dropped upon next release.
`pip3 install pyOpenssl --upgrade`

While troubleshooting errors that were ultimately fixed by the above upgrade, I pinned the `motor` package. I believe a newer version of motor 2.x can be used. There are [breaking changes](https://motor.readthedocs.io/en/stable/changelog.html#breaking-changes) when utilizing motor 3. The hpfeeds team last committed to the repo in March, 2021, so version 2.3.0 was the latest they were likely familiar with.
`pip3 install "motor==2.3.0"`

Following the hpf3 instructions to add a user in mongo. Note, the unauthenticated connection to mongo and the direct interaction. Both can probably be fixed with those handy helper scripts mentioned in #21.
```
mongo
> use hpfeeds
switched to db hpfeeds
> db.auth_key.insert({"identifier": "testing", "secret": "secretkey", "publish": ["chan1", "chan2"], subscribe: ["chan2"]})
WriteResult({ "nInserted" : 1 })
>
```

Running the command line variants of hpf3
`hpfeeds subscribe --host 127.0.0.1 -p 10000 -i testing -s secretkey -c chan2`
(in another shell)
`hpfeeds publish --host 127.0.0.1 -p 10000 -i testing -s secretkey -c chan2 '{"event":"ping"}'`

The last things that need to happen for this install are ensuring pyOpenssl is up to date and motor is using the highest possible (compatible) version.
